app/lib/backend_api_logic/routing_policy.rb

```diff
@@ -73,7 +73,16 @@ def as_json
         def replace_path
           return {} if config_path.blank?
 
-          { replace_path: "{{uri | remove_first: '#{config_path.path}'}}" }
+          { replace_path: "{{uri | remove_first: '#{liquid_path}'}}" }
+        end
+
+        private
+
+        # The replace_path rule is a liquid template. We need to escape some special characters,
+        # otherwise they won't match and won't get removed from the backend uri
+        # The percent sign (%) is also required to be escaped, but we do not accept it in an URI path, so it's OK
+        def liquid_path
+          config_path.path.gsub(/([+-])/, '\1\1')
```

Comment:
:warning: `%` also is needed

Reply:
`%` is not accepted in the path.

```ruby
[1] pry(main)> URI.parse('/my/path').path
=> "/my/path"
[2] pry(main)> URI.parse('/my-path').path
=> "/my-path"
[3] pry(main)> URI.parse('/my+path').path
=> "/my+path"
[4] pry(main)> URI.parse('/my.path').path
=> "/my.path"
[5] pry(main)> URI.parse('/my_path').path
=> "/my_path"
[6] pry(main)> URI.parse('/my%path').path
URI::InvalidURIError: bad URI(is not URI?): /my%path
from /Users/guilherme.cassolato/.rbenv/versions/2.5.1/lib/ruby/2.5.0/uri/rfc3986_parser.rb:67:in `split'
```

Reply:
```
irb(main):001:0> URI.parse('/my%20hello').path
=> "/my%20hello"

```

Reply:
Oops. You are right. I see it now.

Reply:
FYI readers: https://www.w3schools.com/tags/ref_urlencode.ASP